packages/opentelemetry-cloud-trace-exporter/test/transform.test.ts

```diff
@@ -53,6 +53,9 @@ describe('transform', () => {
         cost: 112.12,
       }),
       instrumentationLibrary: {name: 'default', version: '0.0.1'},
+      droppedAttributesCount: 0,
+      droppedEventsCount: 0,
+      droppedLinksCount: 0,
```

Comment:
These fields were added to `ReadableSpan` which is technically a breaking change for anything implementing that interface (structural subtyping)